### DISCUSSION TITLE
Aufgaben aus der Vorlesung die umgesetzt werden müssen/sollten

### DISCUSSION THREAD

**Discussion by Flugschnitzel** [SE-TINF22B2/G4-Get2Gether#22] [Ideas]
**Architektur:**
- Deploymentdiagramm
- Sequenzdiagramm
- Komponentendiagramm

**Comment by Lica200** [comment]
ER-Entwurf: wo ka steht brauchen wir meiner Meinung nach keine Daten, bitte drüberschauen, ob noch Attribute fehlen (wenn jemand herausgefunden hat wie man zusammen an einem VisualParadigm-Projekt arbeiten kann bitte auch Bescheid geben.
![ER_Get2Gether](https://github.com/SE-TINF22B2/G4-Get2Gether/assets/134512122/1bdf9fe1-8851-4f83-92f4-750558775246)

**Reply by Drumber** [reply]
Sieht gut aus, mir sind aber noch paar Dinge aufgefallen:
Das Benutzer Entity hat kein Passwort, da wir den Login über Google abwickeln. Da könnten noch weitere Attribute dazu kommen, sobald wir wissen was wir für den Google Login Flow brauchen. Event_id könnte auch weg, da die Beziehung über Event_Benutzer definiert ist. Das Event braucht auch deswegen kein Teilnehmer Attribut.

In Widget würde ich statt Column eher Row speichern (angenommen wir zeigen die Widgets später untereinander an).

**Comment by Lica200** [comment]
![DatenbankenUebung](https://github.com/SE-TINF22B2/G4-Get2Gether/assets/134512122/6afb61eb-167b-4bf8-8aa4-ca59122b4785)
